OtripleS.Web.Api.Tests.Unit/Services/StudentServiceTests/StudentServiceTests.cs

```diff
@@ -34,7 +34,8 @@ private Student CreateRandomStudent()
         {
             var filler = new Filler<Student>();
             filler.Setup()
-                .OnProperty(student => student.BirthDate).Use(this.dateTimeBroker.GetCurrentDateTime())
+                .OnProperty(student => student.BirthDate)
+                    .Use(this.dateTimeBroker.GetCurrentDateTime().AddYears(-new Random().Next(6,18)))
```

Comment:
Brokers are dependencies - they are meant to be mocked for testing.
In this case here you can simply just generate a random date like this:
```csharp
        private static DateTimeOffset GetRandomDateTime() =>
            new DateTimeRange(earliestDate: new DateTime()).GetValue();
```